beacon-chain/sync/validate_data_column.go

```diff
@@ -48,15 +46,19 @@ func (s *Service) validateDataColumn(ctx context.Context, pid peer.ID, msg *pubs
 	}
 
 	// Ignore messages that are not of the expected type.
-	ds, ok := m.(*eth.DataColumnSidecar)
+	dspb, ok := m.(*eth.DataColumnSidecar)
 	if !ok {
 		log.WithField("message", m).Error("Message is not of type *eth.DataColumnSidecar")
 		return pubsub.ValidationReject, errWrongMessage
 	}
+	ds, err := blocks.NewRODataColumn(dspb)
```

Comment:
Why this variable is name `ds`?
There is no `s` at all in `dataColumns`.
==> `dc`?

Reply:
It just follows from `DataColumnSidecar` where the s refers to the sidecar